src/screens/SearchResultsScreen.tsx

```diff
@@ -20,12 +23,39 @@ export async function loader({request}: {request: Request} ): Promise<string> {
  */
 export default function SearchResultsScreen() {
 	const query: string = useLoaderData() as string;
+	const location = useLocation();
+	const [search, setSearch] = useState(location.search.slice(2));
+	const [data, setData] = useState<MovieData | null>(null);
```

Comment:
```suggestion
	const [movieData, setMovieData] = useState<MovieData | null>(null);
```
nit: could do the same on the same on the state below

Reply:
Feel free to mark this as resolved again. I wasn't sure if it was automatically marked or if you closed it. The suggestion here is to just have more specific var names. If we need to scroll down 50 lines `movieData` will be more clear than `data` especially if we start mixing in user/session data.